README.md

```diff
@@ -18,6 +18,7 @@
 ![.NET MAUI Weather App on all platforms](Assets/maui-weather-hero-sm.png)
 
 ## Current News
+* November 2025 - [What's new in .NET MAUI for .NET 10](https://learn.microsoft.com/dotnet/maui/whats-new/dotnet-10)
```

Comment:
We normally add this link with the GA release in November. The link already exists with previews, but it doesn't make sense to put November here now, as it could cause confusion.

Reply:
Include again the link, but replace "November 2025" to just "2025"